.github/workflows/imageUploadTesting.yml

```diff
@@ -1,24 +1,15 @@
 name: Main-Build .NET API Image and Push to Openshift Registry
 on:
   push:
-    branches: [feature/integration]
+    branches: [feature/workerDeploy]
```

Comment:
We are creating workflows on various different feature branches.  Before we were only building images off of **main**. It would it make sense to explore only building if files were modified in the appropriate directory path?

Reply:
sure. Add comments